src/olympia/activity/urls.py

```diff
@@ -1,16 +1,9 @@
-from django.urls import re_path
+from django.urls import include, re_path
 
 from . import views
 
 
 urlpatterns = [
+    re_path(r'', include('olympia.activity.api_urls')),
     re_path(r'^mail/', views.inbound_email, name='inbound-email-api'),
 ]
-
-attachment_patterns = [
-    re_path(
-        r'^attachment/(?P<log_id>\d+)',
-        views.download_attachment,
-        name='activity.attachment',
-    )
-]
```

Comment:
This change isn't what Mat intended with his request.  

The end result should be 
```python
urlpatterns = [
    re_path(
        r'^attachment/(?P<log_id>\d+)',
        views.download_attachment,
        name='activity.attachment',
    )
]
```
in this file, and 
```python
urlpatterns = [
    re_path(r'^mail/', views.inbound_email, name='inbound-email-api'),
]
```
in api_urls.py.  

You'll then need to change all the `olympia.activity.urls` in https://github.com/mozilla/addons-server/blob/master/src/olympia/api/urls.py to be `olympia.activity.api_urls` instead